daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala

```diff
@@ -55,11 +57,11 @@ object LanguageVersion {
     val textPacking = v1_6
     val enum = v1_6
     val internedPackageId = v1_6
-    val internedStrings = v1_dev
-    val internedDottedNames = v1_dev
-    val numeric = v1_dev
-    val anyType = v1_dev
-    val toTextTypeConName = v1_dev
+    val internedStrings = v1_7
+    val internedDottedNames = v1_7
+    val numeric = v1_7
+    val anyType = v1_7
+    val toTextTypeConName = v1_7
```

Comment:
This change will conflict with #3326.
Once #3326 is merged we will replace by 
```suggestion
    val typeRep = v1_7
```